bench/benchmark.exs

```diff
@@ -0,0 +1,22 @@
+sales_csv = "bench/data/10000_sales.csv"
+sales_string = File.read!(sales_csv)
+sales_list = String.split(sales_string, "\n")
+sales_stream = File.stream!(sales_csv)
```

Comment:
Maybe this should be something like:

```suggestion
sales_stream = Stream.map(sales_list, & &1)
```

So the cost of reading the file is not included in the benchmark?

Reply:
Makes sense, I tried to move reading the file to outside the benchmarks but forgot about the fact that the stream will read it on demand